src/lib/analytics/analytics.js

```diff
@@ -131,7 +131,7 @@ export const reportToSentry = (error, extra = {}, tags = {}) =>
       scope.setTags(key, value)
     })
 
-    Sentry.captureException(error)
+    Sentry.captureException(new Error(errorMsg))
```

Comment:
we log errors in defined convention
from, msg,e.message,e,extra
see how we report in bugsnag

Reply:
@sirpy here I just wrap the error message with Error. It is needed to display the reported issue on sentry.io website correctly. The additional data sends like that: 
![1](https://user-images.githubusercontent.com/49862004/71401550-e0cd6980-2632-11ea-85ee-1379eafafb02.png)